### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by Raymarkjavedalinog** [community/community#134942] [Programming Help]
### Select Topic Area

General

### Body

 I want to learn programming language

**Comment by Sbahdi** [comment]
Type it on YouTube

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Raymarkjavedalinog ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by globaltech316** [comment]
I can help you, @Raymarkjavedalinog 
Which language do you want to learn?

**Comment by ebndev** [comment]
Welcome to GitHub and the GitHub Community @Raymarkjavedalinog 🎆

You are more likely to get a useful response if you share more about what you'd like to accomplish. Giving a few more details might help someone give you a nudge in the right direction. 😄 

We have a few pinned post at the top of the [Programming Help category](https://github.com/orgs/community/discussions/categories/programming-help) that can help you get started as well.
<img width="712" alt="Screenshot 2024-08-07 at 3 39 33 PM" src="https://github.com/user-attachments/assets/47d40b22-68a2-4418-bb99-0421570000df">

Also, feel free to introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋

**Comment by bachmanthomas5** [comment]
Need C programming assignment help? Get expert assistance to tackle complex coding challenges and understand fundamental concepts. Whether you're struggling with loops, pointers, or data structures, reliable support is available to help you complete your assignments with confidence. Reach out today for professional [C programming assignment help](https://www.myassignment-services.com/c-assignment-help.html) and improve your coding skills!

**Comment by MuhamedHabib** [comment]
the  great way is by coding day after day and learning from others experience

**Comment by charlie-paterson** [comment]
I found the best way to learn programming is through practice and tutorials. My favourite tutorials are on linkedin learning, I found these extremely insightful and cover a lot of basics through to more advanced techniques.

**Comment by Asha31s** [comment]
I'd recommend picking a language that has a lot of resources and a supportive community. Python is a great choice for beginners because it's easy to read and has a ton of tutorials available. JavaScript is another good one, especially if you're interested in web development. Once you pick a language, try to work on small projects to practice and get comfortable with coding. Sites like Codecademy, freeCodeCamp, and Coursera can help.
